### PR TITLE
add Dockerfile and build script

### PR DESCRIPTION
**Pull request by mfranczy**
Signed-off-by: Marcin Franczyk <marcin0franczyk@gmail.com>

**What this PR does / why we need it**:
Dockerfile and build script

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```improvement operator
None
```

### CODE REVIEW

File: .ci/build [hunk 1]
```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env bash
+
+# Copyright (c) 2019 SAP SE or an SAP affiliate company. All rights reserved.
```
Comment (moadqassem) [inline .ci/build]: Can you please change the copyright date to 2020?

**Comment by moadqassem** [comment]
@mfranczy can you please add a Makefile target which build the image and push it?

File: .ci/build (hunk 1 above)
Comment (mfranczy) [reply .ci/build]: done

**Comment by mfranczy** [comment]
> @mfranczy can you please add a Makefile target which build the image and push it?

I see you have done this already

**Comment by mfranczy** [comment]
@moadqassem I rebased against master branch.

/cc @stoyanr

**Pull request merged**